### PR TITLE
Adding follow thread message to anonymous thread posts

### LINKED ISSUE #5: feature(ephemeral instructions): After posting message, post ephemeral instructions to follow thread
### Pre-requisites:

- [x] Yes, I looked through both [open and closed issues](../../../issues?utf8=✓&q=is%3Aissue) looking for what I needed

## Feature Description

The ephemeral message should be sent by the bot and should include instructions on how to use Slack's "Follow thread" functionality to be notified since they weren't tagged.

### PR DESCRIPTION
**Pull request by dre8597**
## Pre-Requisites
- [X] Yes, I updated [Authors.md](../Authors.md) **OR** this is not my first contribution
- [X] Yes, I included and/or modified tests to cover relevent code **OR** my change is non-technical
- [X] Yes, I wrote this code entirely myself **OR** I properly attributed these changes in [Third Party Notices](../THIRD-PARTY-NOTICES.txt)

## Description of Changes

Adding an ephemeral message to anonymous questions to notify the user on how to follow a message/thread

Screen Shot 2021-05-13 at 12 48 51 PM

## Related Issues

Closes #5

### CODE REVIEW

File: src/slack/views/postQuestionAnonymouslySubmitted.ts [hunk 1]
```diff
@@ -22,17 +22,27 @@ export const postQuestionAnonymouslySubmitted: Middleware<SlackViewMiddlewareArg
...
+    ack();
 
     await app.client.chat.postMessage({
       token: env.slackToken,
       channel,
       text,
     });
-    ack();
+
+    await app.client.chat.postEphemeral({
```
Comment (SpencerKaiser) [inline src/slack/views/postQuestionAnonymouslySubmitted.ts:33]: Can refactor the other uses in this file with the new pattern for using `client` from the method args?

File: src/slack/views/postQuestionAnonymouslySubmitted.ts [hunk 2]
```diff
@@ -22,17 +22,27 @@ export const postQuestionAnonymouslySubmitted: Middleware<SlackViewMiddlewareArg
...
+      token: env.slackToken,
+      channel,
+      user: body.user.id,
+      text: `Since you won't be notified when someone replies to this question, here are the steps to follow it:
```
Comment (SpencerKaiser) [inline src/slack/views/postQuestionAnonymouslySubmitted.ts:37]:
```suggestion
      text: `:warning: You won't be notified automatically when someone replies to this question. To get notifications from replies, follow the thread:
```

File: src/slack/views/postQuestionAnonymouslySubmitted.ts [hunk 3]
```diff
@@ -22,17 +22,27 @@ export const postQuestionAnonymouslySubmitted: Middleware<SlackViewMiddlewareArg
...
+      1️⃣ Hover over the thread.
+      2️⃣ Click the three dots icon.
+      3️⃣ Select Follow thread to receive notifications for all new thread replies, 
+         or Unfollow thread to stop receiving notifications.`,
```
Comment (SpencerKaiser) [inline src/slack/views/postQuestionAnonymouslySubmitted.ts:41]:
```suggestion
      :one: Hover over the message (mobile app: tap the message)
      :two: Click message options (three dots)
      :three: Select \`Follow thread\``,
```

**Review by SpencerKaiser — CHANGES_REQUESTED** [review]
Quick changes

**Comment by dre8597** [comment]
Closing in favor of [this new PR](https://github.com/AmericanAirlines/AskingForAFriend/compare/pr/dre8597/18?expand=1)

**Pull request closed without merging**